Test fails when using pytest 5.0.*

```
% pytest -x tests/cupy_tests/core_tests/test_raw.py
==================================================================================================================== test session starts =====================================================================================================================
platform linux -- Python 3.6.9, pytest-5.0.1, py-1.8.0, pluggy-0.12.0
rootdir: /path/to/cupy, inifile: setup.cfg
plugins: xdist-1.28.0, forked-1.0.2
collected 7 items                                                                                                                                                                                                                                            

tests/cupy_tests/core_tests/test_raw.py ..F

========================================================================================================================== FAILURES ==========================================================================================================================
_____________________________________________________________________________________________________________ TestRaw.test_get_function_failure ______________________________________________________________________________________________________________

self = <cupy_tests.core_tests.test_raw.TestRaw testMethod=test_get_function_failure>

    def test_get_function_failure(self):
        # in principle this test is better done in test_driver.py, but
        # this error is more likely to appear when using RawModule, so
        # let us do it here
        with pytest.raises(cupy.cuda.driver.CUDADriverError) as ex:
            self.mod2.get_function("no_such_kernel")
>       assert 'CUDA_ERROR_NOT_FOUND' in str(ex)
E       AssertionError: assert 'CUDA_ERROR_NOT_FOUND' in '<ExceptionInfo CUDADriverError tblen=7>'
E        +  where '<ExceptionInfo CUDADriverError tblen=7>' = str(<ExceptionInfo CUDADriverError tblen=7>)

tests/cupy_tests/core_tests/test_raw.py:160: AssertionError
============================================================================================================= 1 failed, 2 passed in 1.02 seconds =============================================================================================================
```

--- a/tests/cupy_tests/core_tests/test_raw.py
+++ b/tests/cupy_tests/core_tests/test_raw.py
@@ -140,7 +140,7 @@ class TestRaw(unittest.TestCase):
     def test_invalid_compiler_flag(self):
         with pytest.raises(cupy.cuda.compiler.CompileException) as ex:
             cupy.RawModule(_test_source3, ("-DPRECISION=3",))
-        assert 'precision not supported' in str(ex)
+        assert 'precision not supported' in str(ex.value)
 
     def test_module_load_failure(self):
         # in principle this test is better done in test_driver.py, but
@@ -149,7 +149,7 @@ class TestRaw(unittest.TestCase):
         import os
         with pytest.raises(cupy.cuda.driver.CUDADriverError) as ex:
             cupy.RawModule(os.path.expanduser("~/this_does_not_exist.cubin"))
-        assert 'CUDA_ERROR_FILE_NOT_FOUND' in str(ex)
+        assert 'CUDA_ERROR_FILE_NOT_FOUND' in str(ex.value)
 
     def test_get_function_failure(self):
         # in principle this test is better done in test_driver.py, but
@@ -157,4 +157,4 @@ class TestRaw(unittest.TestCase):
         # let us do it here
         with pytest.raises(cupy.cuda.driver.CUDADriverError) as ex:
             self.mod2.get_function("no_such_kernel")
-        assert 'CUDA_ERROR_NOT_FOUND' in str(ex)
+        assert 'CUDA_ERROR_NOT_FOUND' in str(ex.value)

